Partial update on SBP-M1 review

Ekaanth opened the pull request.

Ekaanth committed:
initial review update

khssnv committed:
Return back Cargo feature resolver version spec

khssnv committed:
Merge branch 'develop' into feature/MET-72-SBP-M1-review-comments

SBP M1 review comments deleted:

- https://github.com/Metaquity-Network/metaquity-network-node/blob/8c5caf1c864736390be4ca9884f339d79d417ff9/runtime/src/lib.rs#L78

- https://github.com/Metaquity-Network/metaquity-network-node/blob/8c5caf1c864736390be4ca9884f339d79d417ff9/runtime/src/lib.rs#L432

khssnv committed:
Apply rustfmt

Ekaanth commented:
can we remove the comments here? @khssnv

khssnv committed:
Set project version to `0.1.0`

Remove reviewer's comments about resetting the version.

khssnv committed:
Update crates description from 0055edd

Remove reviewer's comments about crates description update.

khssnv commented:
@Ekaanth, I'll remove them one by one updating things as recommended or preserving your updates from [0055edd](https://github.com/Metaquity-Network/metaquity-network-node/pull/23/commits/0055edd1e17dc2c4807a1f3ec6d169f8d2cc61c1).

khssnv committed:
Remove reviewer's comments about the homepage

Homepage URL was added by 0055edd.

khssnv committed:
Add repository URL in workspace Cargo.toml

Remove reviewer's comments about updating repository URL.

khssnv committed:
Add authors and edition to workspace Cargo.toml

Remove reviewer's comments on authors sections.

khssnv committed:
Update Rust compiler version

This version is used for Polkadot v1.0.0 release.

Remove reviewer's comment on toolchain version.

khssnv committed:
Update clap and futures dependencies

Remove reviewer's comment on clap version update.

khssnv committed:
Update node impl name

Remove reviewer's comment on updating the node impl name.

khssnv committed:
Remove reviewer's comment on updating support URL

Support URL already set by 0055edd.

khssnv committed:
Update scale-info version

Remove reviewer's comment on updating the scale-info version.

khssnv committed:
Remove reviewer's comments about Cargo.toml format

khssnv committed:
Autoformat and sort items in manifest files

khssnv committed:
Fix number of decimals in the smallest unit

khssnv committed:
Rename currency unit to MQTY

Remove reviewer's comment about the currency unit name.

khssnv committed:
Use renamed currency units replacing old ones

Ekaanth commented on node/Cargo.toml:
```diff
@@ -1,18 +1,13 @@
-homepage = "https://substrate.io/"
-edition = "2021"
+version = "0.1.0"
+description = "Metaquity Network is the world's first permissioned blockchain and DeFi protocol for Alternative Real World Assets (ARWA)."
```
This can be `Metaquity Network is the world's first permission blockchain and DeFi protocol for Real World Assets (ARWA).` @abhichat85 @ArkoGanguli what do you think?

Ekaanth replied:
As discussed @khssnv update to `Metaquity Network is the World's First Permissioned Blockchain Network and DeFi Protocol for Real World Assets (RWA).`

khssnv replied:
Done

khssnv committed:
Update node description

khssnv committed:
Set block time to 12 seconds
